packages/core/src/browser/addEventListener.spec.ts

```diff
@@ -35,6 +36,54 @@ describe('addEventListener', () => {
     })
   })
 
+  it('Use the EventTarget.prototype.addEventListener when the eventTarget is an instance of EventTarget', () => {
+    if (isIE()) {
+      pending('EventTarget not supported in IE')
+    }
+
+    const addEventListenerSpy = jasmine.createSpy()
+    const removeEventListenerSpy = jasmine.createSpy()
+
+    // The  stopLeakDetection function of the global after each hook will reset the EventTarget.prototype.addEventListener
```

Comment:
The addEventListener override conflicts with the [leakDetection](https://github.com/DataDog/browser-sdk/blob/179a8a6e7ffb5b37d9ac71e43bab999e1997f4fc/packages/core/test/forEach.spec.ts#L11). the best way I found it to let the [stopLeakDetection](https://github.com/DataDog/browser-sdk/blob/179a8a6e7ffb5b37d9ac71e43bab999e1997f4fc/packages/core/test/forEach.spec.ts#L19) remove the override

Reply:
Okay I found the root cause. Once this [fix](https://github.com/DataDog/browser-sdk/pull/3141) is merged I'll cleanup the tests properly